DigitalOffice/Services/UserService/Validators/UserLoginValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UserService.Models;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class UserLoginValidator : AbstractValidator<UserCredentials>
+    {
+        public UserLoginValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .MaximumLength(50)
+                .EmailAddress();
+            RuleFor(user => user.Password)
+                .NotEmpty();
+        }
```

Comment:
I think, there should be some messages for "breaking" the rules. For each of them.
Also, password should have MinimumLength like 6 or 8 symbols (check Denis's Registration branch, I've seen some limitations for it).

Reply:
Fixed, although Denis's registration branch doesn't include any password checks except for emptiness check

Reply:
Okay, thx, so I'll be waiting for the tests :)

Reply:
fixed